src/main/java/com/google/sps/servlets/ListOrganizationsServlet.java

```diff
@@ -87,11 +97,10 @@ public Query getQueryFromParams(HttpServletRequest request, GivrUser currentUser
       query.setFilter(new FilterPredicate("moderatorList", FilterOperator.EQUAL, userId));
     }
 
```

Comment:
The only change here was uncommenting out this code now that indexing is working*